src/save.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * Internal dependencies
+ */
 import Preview from './preview';
 
 export default function Save( { attributes } ) {
```

Comment:
@cdevroe Two questions here:

1. How can we pass any extra custom HTML attributes (like class names) to the block element wrapper?
2. How to ensure that the block class name is rendered properly?

Reply:
> 1. How can we pass any extra custom HTML attributes (like class names) to the block element wrapper?

Good catch. Sorry. Pulling in the block properties will take any custom classes added in the editor and apply them to the wrapping DIV. I’ve added the code for this to the PR.

> 2. How to ensure that the block class name is rendered properly?

I’m sorry, I don't fully understand this question. 🤔